gwdetchar/scattering/core.py

```diff
@@ -77,6 +76,9 @@
     'TMSY': ['SUS-TMSY_M1_DAMP_L_IN1_DQ'],
 }
 
+TRANSMON_CHANNELS = ['L1:ASC-Y_TR_B_NSUM_OUT_DQ',
+                     'L1:ASC-X_TR_B_NSUM_OUT_DQ']
```

Comment:
@siddharth101, can you remove the IFO prefix from these channels? i.e., remove the `L1:` (these need to work at Hanford as well).